prometheus-exporters/ucs-exporter/values.yaml

```diff
@@ -0,0 +1,21 @@
+ucs:
+   enabled: false
+   image:
+      name: ucs-exporter
+      tag: DEFINED-IN-REGION-SECRETS
+   port_number: 9876
+   alerts:
+      enabled: false
+      prometheus: DEFINED-IN-REGION-SECRETS
+   user: DEFINED-IN-REGION-SECRETS
+   master_password: DEFINED-IN-REGION-SECRETS
+   inventory: "inventory"
```

Comment:
what is this?

Reply:
In program we need to get the inventory file name with -i flag, so I provided it here, not sure if its correct way. pls suggest.

Reply:
This need to be the path then, where the inventory file sits. Please contact me to get this sorted.